### PR TITLE
[BUGFIX] Default string, this should never appear #2703

### LINKED ISSUE #2703: [BUG] 'Default string, this should never appear'
**Type**: Code - general bugs related to how the game runs and responds to you. 

**Describe the bug**
A moon relationship event gave a default string that notes that it should never be seen. Uncertain exactly what happened, the command window error only says 'ERROR: No interaction with this conditions.  platonic decrease high'

**Grade**: Non-vital - This bug can easily be ignored and does not have a large impact on play

**Reproduce**
1. Start a clan
2. Roll moon events & check the relationships tab

**Patrol ID + Outcome Print (if applicable)**: N/A

**Commit # or Game Version Number**: Commit  #2690

**Additional context**
Checking the events.json file, the cats involved in the interaction are Orchidroot (warrior/thoughtful/good speaker & talented swimmer/has an apprentice/adult/clan founder) and Shadebellow (elder/fierce/good climber/senior/clan founder/has had water in her lungs for one moon). Very little else of note has happened, the clan is only 2 moons old.

Uncertain which traits here would be causing the event to fail and use the default string, but hopefully this narrows it down a little

**Screenshots**
image
image

### PR DESCRIPTION
**Pull request by PoppyBlossom**
## About The Pull Request
This pull request addresses an issue where the game lacked an unrestricted "dislike high" interaction after the only nonrestrictive "dislike high" was demoted to medium in #2690. So I added a few more interactions to compensate for this. The error message for no interaction was changed from an error to a warning, and the default string interaction was removed since this issue doesn't significantly impact gameplay.

## Linked Issues
fixes: #2703

## Proof of Testing
The issue no longer occurs and I ran for an additional 200 moons just to make sure.

## Changelog/Credits
Fixed default string.
_Can I be added to the credits list by the way_

### CODE REVIEW

**Comment by PoppyBlossom** [comment]
Look what I found!!!
image
**_DISCLAMER: I did this just for fun, and out of curiosity_**

File: resources/dicts/relationship_events/normal_interactions/platonic/decrease.json [hunk 1]
```diff
@@ -179,5 +176,20 @@
...
+	},
+	{
+		"id": "platonic_de_high4",
+		"intensity": "high",
+		"interactions": [
+			"r_c is caught tossing out a gift m_c <i>just</i> gave {PRONOUN/m_c/object}.",
+			"m_c caught r_c complaining about {PRONOUN/m_c/object} behind {PRONOUN/m_c/poss} back.",
+			"m_c interrupted r_c when {PRONOUN/r_c/subject} {VERB/r_c/was/were} very busy with an important task, ruining {PRONOUN/r_c/poss} focus.",
+			"r_c purposefully pick the smallest prey for m_c when {PRONOUN/m_c/subject} asked for help.",
+			"r_c was find spreading rumors about m_c stealing stuff from {PRONOUN/r_c/object}.",
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/platonic/decrease.json]:
```suggestion
			"r_c was found spreading rumors about m_c stealing stuff from {PRONOUN/r_c/object}.",
```

File: resources/dicts/relationship_events/normal_interactions/platonic/decrease.json [hunk 2]
```diff
@@ -179,5 +176,20 @@
...
+			"r_c was caught hiding m_c belongings.",
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/platonic/decrease.json]:
```suggestion
			"r_c was caught hiding m_c's belongings.",
```

**Review by scribblecrumb — CHANGES_REQUESTED** [review]

**Comment by scribblecrumb** [comment]
Feel free to add yourself to the credits_text.json!

**Review by scribblecrumb — APPROVED** [review]

**Pull request merged**